src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -1,5 +1,19 @@
 package mate.academy
 
+import java.util.*
```

Comment:
FYI in similar cases try to use imports which are used in the code only. Import via 'star' could be not optimal